gemrb/core/System/VFS.h

```diff
@@ -126,6 +126,11 @@ GEM_EXPORT char* CopyHomePath(char* outPath, ieWord maxLen);
 // default directory housing GUIScripts/Override/Unhardcoded
 GEM_EXPORT char* CopyGemDataPath(char* outPath, ieWord maxLen);
 
+void* readonly_mmap(void *fd);
+#ifdef WIN32
+void munmap(void *start, size_t);
+#endif
```

Comment:
Does this work on mingw too or does it cause a redefinition error? Wondering if we need to test munmap presence in cmake.

Reply:
I'm not really sure how much `-G "MinGW Makefiles"` inside of an MSYS2 shell is the answer here, but I got no errors when doing `mingw32-make` . And from googling, it looks as if you still need a wrapper anyway.

Reply:
Ok, we'll deal with it later if someone manages to break it.